src/server/forum_user_handler.cpp

```diff
@@ -406,6 +408,15 @@ void fuh::write_detail(const std::string& name, const std::string& detail, T&& v
 	}
 }
 
+bool fuh::is_user_in_group(const std::string& name, unsigned int group_id) {
+	try {
+		return prepared_statement<bool>("SELECT 1 FROM `" + db_users_table_ + "` u, `" + db_group_table_ + "` ug where UPPER(u.username)=UPPER(?) and u.USER_ID = ug.USER_ID and ug.GROUP_ID = ?", name, group_id);
```

Comment:
If we want to stick with uppercasing all SQL keywords then where and and should be uppercase as well.